orchagent/fdborch.cpp

```diff
@@ -39,8 +44,9 @@ FdbOrch::FdbOrch(DBConnector* applDbConnector, vector<table_name_with_pri_t> app
     Orch::addExecutor(flushNotifier);
 
     /* Add FDB notifications support from ASIC */
-    DBConnector *notificationsDb = new DBConnector("ASIC_DB", 0);
-    m_fdbNotificationConsumer = new swss::NotificationConsumer(notificationsDb, "NOTIFICATIONS");
+       DBConnector *notificationsDb = new DBConnector("ASIC_DB", 0);
```

Comment:
Please fix alignment

Reply:
@prsunny : Thanks for the comment. Done.